controller/rancher/rancher_events_handler.go

```diff
@@ -0,0 +1,243 @@
+package rancher
+
+import (
+	"fmt"
+	"github.com/Sirupsen/logrus"
+	revents "github.com/rancher/event-subscriber/events"
+	"github.com/rancher/go-rancher/v2"
+	"github.com/rancher/lb-controller/config"
+	"sync"
+	"time"
+)
+
+type EventsHandler interface {
```

Comment:
No need to change this, but is there a reason you created an interface here? I don't really see what purpose it serves. I tend to avoid interfaces in golang unless the serve an explicit immediate purpose.

Reply:
If you're using this for testing, my apologies for not seeing that

Reply:
I was planning to use it for tests, but have not added events level tests.